src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -446,6 +446,23 @@ class ActivityFeed extends React.Component<Props, State> {
             feedItems.push(...itemContainer.entries);
         });
 
+        const { file } = this.props;
+        const { restored_from, modified_at } = file;
+        if (restored_from) {
+            const restoredVersion = feedItems.find(
+                (feedItem) => feedItem.type === 'file_version' && feedItem.id === restored_from.id
+            );
+
+            if (restoredVersion) {
```

Comment:
will this still work if the versions in here would normally be grouped, but now we're learning that there's a restored version in here? I feel like this should happen at the grouping stage.